save-backend/src/main/kotlin/com/saveourtool/save/backend/storage/TestSuitesSourceSnapshotStorage.kt

```diff
@@ -64,6 +64,23 @@ class TestSuitesSourceSnapshotStorage(
         .map { true }
         .defaultIfEmpty(false)
 
+    /**
+     * @param organizationName
+     * @param testSuitesSourceName
+     * @param version
+     * @return true if backend removes corresponding test suite source
+     */
+    fun removeKey(
+        organizationName: String,
+        testSuitesSourceName: String,
+        version: String,
+    ): Mono<Boolean> = findKey(organizationName, testSuitesSourceName, version)
+        .flatMap { key ->
+            delete(key).map { isRemoved ->
+                isRemoved
+            }
```

Comment:
```suggestion
            delete(key)
```
Is this `.map` needed?

Reply:
Thanks, removed